spec/support/tenant_identity.rb

```diff
@@ -0,0 +1,15 @@
+module Spec
+  module Support
+    module TenantIdentity
+      extend ActiveSupport::Concern
+
+      included do
+        let!(:tenant)           { Tenant.create!(:external_tenant => external_tenant) }
+        let!(:external_tenant)  { rand(1000).to_s }
+        let!(:unknown_tenant)   { rand(1000).to_s }
+        let!(:identity)         { Base64.encode64({'identity' => { 'account_number' => external_tenant}}.to_json) }
+        let!(:unknown_identity) { Base64.encode64({'identity' => { 'account_number' => unknown_tenant}}.to_json) }
```

Comment:
Feels like we should have helper methods in ManageIQ::API::Common for this...future follow up. cc @syncrou

Reply:
@Fryguy, @gtanzillo  - Yes.  We actually already do.  https://github.com/ManageIQ/manageiq-api-common/blob/master/spec/support/user_header_spec_helper.rb

Reply:
There are a number of examples in the manageiq-common and the catalog where a new key/value is applied and re Base64 encoded as well.